labelbox/schema/project.py

```diff
@@ -166,7 +167,14 @@ def export_queued_data_rows(self, timeout_seconds=120):
                 self.uid)
             time.sleep(sleep_time)
 
-    def export_labels(self, timeout_seconds=60):
+    def export_labels(self):
+        json_data = self.export_labels_json()
+        if 'frames' in json_data[0]['Label']:
+            # Assumes all data rows in this project are video.
```

Comment:
This seems like a stretch of an assumption. Maybe add a docstring that mentions this?

Reply:
Yeah I think I am going to create a separate function for exporting video.